### PR TITLE
Add more folders

### PR DESCRIPTION
**Pull request by OtmaroE**
Please let me add more folders

### CODE REVIEW

**Comment by erickv94** [comment]
Hello

File: index.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
-const output = () => console.log('This is an output');
+const consoleOutput = () => console.log('This is an output');
```
Comment (sotnasaj) [inline index.js:1]: Comment

Comment (Agustin07) [inline index.js:1]: text comment!

Comment (OtmaroE) [inline index.js:1]: This name does not work

**Pull request merged**